nml/version_info.py

```diff
@@ -135,15 +135,23 @@ def get_cli_version():
     return result[0:-1] #strip trailing newline
 
 def get_and_write_version():
+    try:
+        from nml import __version__
+        version = __version__.version
+        if __version__.release:
+            return version
+    except ImportError:
+        pass
     version = get_nml_version()
     if version:
         try:
             path = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
             f = open(os.path.join(path, "nml", "__version__.py"), "w")
             f.write('# this file is autogenerated by setup.py\n')
             f.write('version = "{}"\n'.format(version))
+            f.write('release = {}\n'.format(os.getenv("GITHUB_ACTIONS") != None))
             f.close()
-            return version.split()[0]
+            return version
```

Comment:
Why is this needed? Is this just som old code from when version included more than just a single version and is this just a cleanup?